src/utils/test/metrics_test.cpp

```diff
@@ -3045,11 +3045,11 @@ TEST_P(MetricsRetirementTest, RetireOldMetrics)
 }
 
 const std::vector<surviving_metrics_case> metrics_retirement_tests = {
-    {"server_117", true, true, true, true},
-    {"server_118", true, true, true, false},
-    {"server_119", true, true, false, false},
-    {"server_120", true, false, false, false},
-    {"server_121", false, false, false, false},
+    {std::string("server_117"), true, true, true, true},
```

Comment:
Does this fix https://github.com/apache/incubator-pegasus/issues/1424 for ubuntu1604 ?

Reply:
Yep，should I recover it？